src/app/sidenav/sidenav.component.ts

```diff
@@ -8,6 +8,10 @@ import {MatSidenav} from '@angular/material';
 })
 export class SidenavComponent implements OnInit {
   @ViewChild('sidenav') sidenav: MatSidenav;
+  isFavorited = false;
+  communityName: String;
```

Comment:
This doesn't need to be an object property since you only use it in the `ngOniInit` function.